theano/misc/pkl_utils.py

```diff
@@ -4,11 +4,31 @@
 These pickled graphs can be used, for instance, as cases for
 unit tests or regression tests.
 """
+import numpy
 import pickle
 import sys
+import tempfile
+import zipfile
+import warnings
+from collections import defaultdict
+from contextlib import closing
+from pickle import HIGHEST_PROTOCOL
+from theano.compat.six import StringIO
+try:
+    from pickle import DEFAULT_PROTOCOL
+except ImportError:
+    DEFAULT_PROTOCOL = HIGHEST_PROTOCOL
+
 import theano
+from theano import config
 from theano.compat import PY3
 from theano.compat.six import string_types
+from theano.compile.sharedvalue import SharedVariable
+try:
+    from cuda_ndarray import cuda_ndarray
```

Comment:
How this can work? I think you need to do:

from theano.sandbox.cuda.cuda_ndarray import cuda_ndarray